modules/configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery"]
+= Setting up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover VNet and subnet names
+
+The following procedure shows you how to set up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover VNet and subnet names.
+
+.Prerequisites 
+
+* You have configured an image registry to run on Azure.
+* Your network has been set up using the Installer Provisioned Infrastructure (IPI) installation method. 
++
+For users with VNet and subnet names, see "Configuring a private storage endpoint on Azure with user-provided VNet and subnet names". 
```

Comment:
Nice!
Is there anyway `"Configuring a private storage endpoint on Azure with user-provided VNet and subnet names"` can be a link that points to that procedure?

Reply:
@flavianmissi Short answer is no. Longer answer: It's a docs limitation. We can't use cross references in modules because it'll break the build on Pantheon. I might be able to use a direct HTTP link, but we try not to do that generally. I'll have to check out guidelines. We generally rely on users using CTRL+F (not a great docs experience). Hoping this is resolved when we get the new toolchain.

Reply:
Aha, thanks for the explanation!